client/app/components/dashboards/lazyInjector.js

```diff
@@ -0,0 +1,16 @@
+let $injector;
+
+const lazyInjector = {
+  get $injector() {
+    return {
+      get get() {
+        return $injector.get;
+      },
+    };
+  },
+  set $injector(_$injector) {
+    $injector = _$injector;
+  },
+};
+
+export default lazyInjector;
```

Comment:
I think there is a room for improvements:
1. move `export default function init(ngModule)` + `ngModule.run(...)` to this file and init `$injector` global here as well;
2. remove setter from `lazyInjector`;
3. simplify it:
```js
export const $injector = {
  get get() {
     return $injector.get;
  };
}

// usage:
import { $injector } from './lazyInjector';
export ... = anrgular2react(..., ..., $injector);
```
4. move this code to `@/services/ng`

Reply:
I tried.. but couldn't make it work. Don't wanna invest more time in it (especially knowing this will be soon removed).
But feel free to push an update if it comes easy to you.